features/checkout/shipping_order/preventing_placing_order_with_disabled_shipping_method.feature

```diff
@@ -0,0 +1,20 @@
+@checkout
+Feature: Preventing placing an order with a disabled shipping method
+    In order to ship my order properly
```

Comment:
```suggestion
    In order to have my order processed smoothly
```
or
```suggestion
    In order to not provide invalid data for my order
```
or
```suggestion
    In order to have my order shipped without issues
```
or something similar, we're in the checkout context so the original doesn't make any sense.